application/modules/statements/models/Mdl_statement.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+class Mdl_Statement extends CI_Model {
```

Comment:
@pradeshc Please try to document your classes, variables and functions with proper types and descriptions. Also, please add the copyright notice.

Reply:
Done